server/src/cm_server_interface.cpp

```diff
@@ -593,6 +593,7 @@ cm_execute_request_async (Json::Value &request, Json::Value &response,
                                   "failed to run task.");
     }
 
+  time_out = 30;
```

Comment:
The value of the time_out variable is determined by sco.iHttpTimeout in cm_server_interface.cpp line 753.
And, the sco.iHttpTimeout value is initialized at line 238 of cm_config.cpp. If you change the initial value from cm_config.cpp to 30, linux/windows will all work the same. What do you think about?

Reply:
sco.iHttpTimeout is currently set to 500 seconds.
The reason I did not change the sco.iHttpTimeout is because I do not want to affect other behaviors of cub_manager except cm_execute_request_async ().

Reply:
I think you right! 
I changed sco.iHttpTimeout to 30 instead temporarily change t/o to 30 in cm_execute_request_async ()
Thank you.